dataset/dictionary/items.json5

```
@@ -1235,6 +1235,14 @@
     pronunciationJa: "びゃくやこくちりすいもんし", // TODO Need check
     tags: [ "item", "inazuma" ],
   },
+  {
+    en: "Festive Drum",
+    ja: "綺宴の鼓",
+    zhCN: "绮筵之鼓",
+    pronunciationJa: "きえんのこ",  // きえんのつづみ? TODO Need check
```

Just a stylistic fix, removed double whitespaces.

```suggestion
    pronunciationJa: "きえんのこ", // きえんのつづみ? TODO Need check
```